deploy/kubernetes/overlays/dev/controller_images.yaml

```diff
@@ -6,6 +6,12 @@ spec:
   template:
     spec:
       containers:
+        - name: csi-provisioner
```

Comment:
Separately we should also consider if we want a job that tests latest sidecars too

Reply:
prow-gke-head does that